### PR TITLE
Adds load_from_data function to CssProvider

### PR DESCRIPTION
**Pull request by honorabrutroll**
Fairly simple addition (almost exactly the same as load_from_path) and has worked in my tests.
[Progress Bar Test](https://github.com/honorabrutroll/gtk-dl-poc/blob/master/src/gui.rs)

### CODE REVIEW

**Comment by gkoz** [comment]
Looks good to me, thanks!
:+1:

File: src/widgets/css_provider.rs [hunk 1]
```diff
@@ -33,6 +34,21 @@ impl CssProvider {
...
+        unsafe {
+            let pointer = ffi::gtk_css_provider_new();
+            let mut error = ::std::ptr::null_mut();
+            ffi::gtk_css_provider_load_from_data(pointer, data.as_ptr() as *mut u8, data.len() as ssize_t, &mut error);
+            if error.is_null() {
+                Ok(CssProvider { pointer: pointer })
+            } else {
+                Err(glib::Error::wrap(error))
+            }
+
```
Comment (GuillaumeGomez) [inline src/widgets/css_provider.rs]: This empty line should be removed.

**Comment by GuillaumeGomez** [comment]
Hi, thanks for your work! I wait for you to fix what I commented and then I merge.

**Comment by honorabrutroll** [comment]
Would you also like for me to remove the empty line from ```load_from_path```? That's originally where the idea for the line came from and I assumed that it was there for stylistic purposes or something.

**Comment by GuillaumeGomez** [comment]
Yes please! :)

**Comment by gkoz** [comment]
Alright, this one seems ready.

**Pull request merged**